xarray/plot/plot.py

```diff
@@ -941,3 +1013,82 @@ def pcolormesh(x, y, z, ax, infer_intervals=None, **kwargs):
         ax.set_ylim(y[0], y[-1])
 
     return primitive
+
+
+def scatter(ds, x, y, hue=None, col=None, row=None,
```

Comment:
Move this and associated functions to `dataset_plot.py`?

Reply:
I thought about that. But for consistency we should then rename `plot.py` to `DataArray_plot.py`, right?